jsonnet/openebs-mixin/rules/npd-rules.libsonnet

```diff
@@ -1,8 +1,8 @@
 function(param) {
-  local rules = self,
+  //local rules = self,
```

Comment:
Why is this commented?

Reply:
For now, we are not using any configuration in defining the rules. That's why it is not required.